### PR TITLE
Fixes #17227: Prevent removal of 'system' global and group properties

### PR DESCRIPTION
**Pull request by fanf**
https://issues.rudder.io/issues/17227

### CODE REVIEW

**Comment by VinceMacBuche** [comment]
I think you need to rebase and remove first commit!

File: webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/marshalling/XmlSerialisation.scala [hunk 1]
```diff
@@ -346,7 +346,7 @@ trait GlobalParameterSerialisation {
        <name>{param.name.value}</name>
        <value>{param.value}</value>
        <description>{param.description}</description>
-       <overridable>{param.overridable}</overridable>
+       <provider>{param.overridable}</provider>
```
Comment (VinceMacBuche) [inline webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/marshalling/XmlSerialisation.scala:349]: param.provider

**Comment by fanf** [comment]
PR rebased

**Comment by fanf** [comment]
PR updated with a new commit

**Comment by fanf** [comment]
OK, squash merging this PR

**Pull request merged**